rotkehlchen/externalapis/gnosispay.py

```diff
@@ -113,8 +114,10 @@ def _query(
 
         if response.status_code != HTTPStatus.OK:
             if response.status_code == HTTPStatus.UNAUTHORIZED:
-                self.database.msg_aggregator.add_warning(
-                    msg='Gnosis Pay authentication token is invalid or has expired. Either remove it or get a new one.',  # noqa: E501
+                error_message = response.json().get('message', response.text)
```

Comment:
the .json can raise a exception from `requests.exceptions` JsonDecodeError irc. It needs to be handled